sympy/geometry/tests/test_line.py

```diff
@@ -54,13 +54,13 @@ def test_line_geom():
     raises(ValueError, lambda: Line(Point(0, 0), Point(0, 0)))
 
     # Basic stuff
-    assert Line((1, 1), slope=1) == Line((1, 1), (2, 2))
```

Comment:
I would spend time on it if its something that I had to look at a lot. But I don't so I wouldn't. The most important thing is that the code if covered in a meaningful way. Beautifying the test suite is low on my list of things to do. But it may be meaningful to you if it is an entry way to better understanding the geometry module and I would be willing to review whatever you do.